### DISCUSSION TITLE
Fast Panel Integration

### DISCUSSION THREAD

**Discussion by 021fneves** [Paymenter/Paymenter#365] [Ideas]
Would integration with Fast Panel be possible?
https://fastpanel.direct

**Comment by SirHoDo** [comment]
Considering their API has been "coming soon" on github for 3 years, I don't know how possible it is. Though if you can site sources for API, I am sure someone might eventually

**Reply by 021fneves** [reply]
Got it, I didn't find Fast Panel API either.

**Comment by finnjosten** [comment]
I would suggest just to use froxlor

They have a working api so you could make a addon

**Reply by 021fneves** [reply]
I've already looked at Froxlor, but I think it still needs to be a little more simplified for the end user to add a domain.
I also looked at CyberPanel, I didn't like it very much.

**Reply by finnjosten** [reply]
Yeah domains can still have some work but overall it's very good. Of all its the safest bet I would say as its clean and pretty decent with an api

**Reply by 021fneves** [reply]
Thanks!
I'll take a better look at Froxlor

**Reply by finnjosten** [reply]
Yeah, no use what you like of course since well personal preference.
